astropy/visualization/wcsaxes/tests/test_coordinate_helpers.py

```diff
@@ -163,3 +164,19 @@ def test_get_position():
     assert ax.coords[1].get_ticklabel_position() == ["r", "l"]
     assert ax.coords[0].get_axislabel_position() == ["t"]
     assert ax.coords[1].get_axislabel_position() == ["r"]
+
+
+def test_deprecated_getters():
+    fig, _ = plt.subplots()
+    ax = WCSAxes(fig, [0.1, 0.1, 0.8, 0.8], aspect="equal")
+    helper = CoordinateHelper(parent_axes=ax)
+
+    with pytest.warns(AstropyDeprecationWarning):
+        ticks = helper.ticks
+    assert not ticks.get_display_minor_ticks()
+    with pytest.warns(AstropyDeprecationWarning):
+        ticklabels = helper.ticklabels
+    assert ticklabels.text == {}
```

Comment:
Just to be thorough: I just checked how this tested behaved when run without the companion patch, and this assert is the only one that fails. Is it what you expect ?

Reply:
Yeah, that's what I would expect. I was following Tom's suggestion of testing the three deprecated getters, but only the `ticklabels` getter is broken right now. The checks themselves aren't particularly meaningful beyond the fact that if the getter returns the wrong member (as happens with `ticklabels`) the statement should fail.